src/services/types.ts

```diff
@@ -199,6 +202,9 @@ namespace ts {
          * Gets a set of custom transformers to use during emit.
          */
         getCustomTransformers?(): CustomTransformers | undefined;
+
+        tryGetTypesRegistry?(): Map<void> | undefined;

```

Comment:
consider changing the API to be `isKownTypesPackageName(name: string)`. the server still gets the registry internally and uses it to answer these requests. but the LS API does not really need to know about this detail.